test-helpers/src/cert.rs

```diff
@@ -34,3 +37,17 @@ pub fn generate_test_certs(path: &Path) {
     .unwrap();
     std::fs::write(path.join("redis.key"), cert.serialize_private_key_pem()).unwrap();
 }
+
+pub fn generate_cassandra_test_certs() {
+    let current_path = current_dir().unwrap();
+    set_current_dir("example-configs/cassandra-tls/certs").unwrap();
```

Comment:
If we panic in the middle that will leave other tests with a different working directory right?
Can this be written such that it doesnt change the directory?
Maybe the script should be responsible for settings its own directory?

TBH I would rather we just rewrite the script entirely using rust, I would need some convincing that the existing `generate_test_certs` can not be adapted for cassandra.
That said this stuff can be a bit complicated so im willing to compromise here for the sake of moving things forward.